pkg/datapath/loader/cache.go

```diff
@@ -65,6 +65,17 @@ func Init(dp datapath.Datapath, nodeCfg *datapath.LocalNodeConfiguration) {
 	elfCache.Update(nodeCfg)
 }
 
+// RestoreTemplates populates the object cache from templates on the filesystem
+// at the specified path.
+func RestoreTemplates(stateDir string) error {
```

Comment:
Is there a plan to restore these eventually?

Reply:
Yep, that's why it's called RestoreTemplates. I figured this is unnecessary to get the basics working, but it should be a trivial patch on top.

Reply:
Awesome :D